### PR TITLE
Release 2.7.0

### LINKED ISSUE #339: [question]no ep_rewmean, eplenmean displayed, I don't know why
**Describe the bug**
when run my program,the fallowing displayed, but no ep_rewmean, eplenmean displayed, I don't know why
//-------------------------------------
| approxkl           | 0.0          |
| clipfrac           | 0.0          |
| explained_variance | -5.96e-07    |
| fps                | 49           |
| nupdates           | 77746        |
| policy_entropy     | 6.521396e-05 |
| policy_loss        | 0.0          |
| serial_timesteps   | 2487872      |
| time_elapsed       | 4.58e+04     |
| total_timesteps    | 1166190      |
| value_loss         | 278.51614    |
//-------------------------------------

**Code example**
model = PPO2(MlpLstmPolicy, train_env,n_steps=32, nminibatches=1,lam=0.95, gamma=0.99, noptepochs=15,
             vf_coef = 0.0001,ent_coef=0.0001,learning_rate=0.00005,cliprange=0.2,verbose=1,
            tensorboard_log="./tensorboard/")

**System Info**
Describe the characteristic of your environment:
 * Describe how the library was installed (pip, docker, source, ...)
pip
 * GPU models and configuration
gpu
 * Python version
3.6
 * Tensorflow version
1.13
 * Versions of any other relevant libraries

**Additional context**
the ep_rewmean also not displayed in tensorboard, I also want the ep_rewmean displayed in tensorboard. Could you know if there is any way?

### LINKED ISSUE #379: [question] PPO2 fps= 0
I'm testing ppo2 using an implementation that uses openai and airsim. The PPO fps shows 0 all the time.

```| approxkl           | 0.011864   |
| clipfrac           | 0.203125   |
| ep_len_mean        | 47.5       |
| ep_reward_mean     | -160       |
| explained_variance | -0.00741   |
| fps                | 0          |
| n_updates          | 1          |
| policy_entropy     | 1.78051    |
| policy_loss        | -0.0219748 |
| serial_timesteps   | 128        |
| time_elapsed       | 0          |
| total_timesteps    | 128        |
| value_loss         | 1113.06    |
-----------------------------------
```
This is how I'm calling ppo 
```
....

# Get the environment and extract the number of actions.
env = gym.make(args.env_name)
env = Monitor(env, log_dir, allow_early_resets=True)
np.random.seed(123)
env.seed(123)
 
nb_actions = env.action_space.n

env = DummyVecEnv([lambda: env])  # The algorithms require a vectorized environment to run
env = VecFrameStack(env, n_stack=4)

....

model = PPO2(CnnPolicy, env,nminibatches = 4, verbose=1,tensorboard_log=log_dir)
....
```
Appreciate your thoughts! 

Thanks

### PR DESCRIPTION
**Pull request by araffin**
Closes #339 and addresses #409

### CODE REVIEW

File: docs/guide/tensorboard.rst [hunk 1]
```diff
@@ -76,6 +76,42 @@ It will display information such as the model graph, the episode reward, the mod
   :width: 400
   :alt: graph
 
+
+Logging More Values
+-------------------
+
+Using a callback, you can easily log more values with tensorboard.
```
Comment (AdamGleave) [inline docs/guide/tensorboard.rst]: s/tensorboard/TensorBoard

File: docs/misc/changelog.rst [hunk 2]
```diff
@@ -6,29 +6,28 @@ Changelog
 For download links, please look at `Github release page <https://github.com/hill-a/stable-baselines/releases>`_.
 
 
-Pre-Release 2.7.0a0 (WIP)
+Pre-Release 2.7.0 (WIP)
```
Comment (AdamGleave) [inline docs/misc/changelog.rst]: This should be changed to Release 2.7.0 and drop the WIP before we release?

Comment (araffin) [reply docs/misc/changelog.rst]: Nice catch

File: docs/guide/tensorboard.rst (hunk 1 above)
Comment (araffin) [reply docs/guide/tensorboard.rst]: You mean capitalize TensorBoard?

**Comment by araffin** [comment]
>I think you mean closes #339? 

Yes, thanks for the review, it was definitely too late yesterday ...

File: docs/guide/tensorboard.rst (hunk 1 above)
Comment (AdamGleave) [reply docs/guide/tensorboard.rst]: Yes, that's what I meant.

**Pull request merged**